target/linux/lantiq/base-files/etc/board.d/02_network

```diff
@@ -152,7 +152,20 @@ WBMR300)
 	ucidef_add_switch "switch0" \
 		"2:lan:1" "3:lan:2" "5:lan:3" "4:wan:1" "6t@eth0"
 	;;
-
+RS353)
+        ucidef_add_switch "switch0" \
+                "4:lan:1" "2:lan:2" "1:lan:3" "0:lan:4" "5:lan:5" "6t@eth0"
+	# Little hack to get the macaddr from flash and reorder bytes
```

Comment:
1. Please use tabs instead of spaces.
2. drop this hackery. use mtd-mac-address instead of ```mac-address = [ 00 11 22 33 44 55 ];``` in the dts. have a look at the other dts files.
3. drop the vpi, vci, annex, encaps and payload params. we already ship a default config at the beginning of the file.